sdk/nodejs/runtime/resource.ts

```diff
@@ -501,6 +503,38 @@ export function registerResourceOutputs(res: Resource, outputs: Inputs | Promise
     }, false);
 }
 
+function isAny(o: any): o is any {
+    return true;
+}
+
+/**
+ * listResourceOutputs returns the resource outputs (if any) for a stack, or an error if the stack
+ * cannot be found. Resources are retrieved from the latest stack snapshot, which may include
+ * ongoing updates.
+ *
+ * @param stackName Name of stack to retrieve resource outputs for. Defaults to the current stack.
+ * @param typeFilter A [type
+ * guard](https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards)
+ * that specifies which resource types to list outputs of.
+ *
+ * @example pulumi.runtime.listResourceOutput(aws.s3.Bucket.isInstance)
```

Comment:
this is actually an incorrect example, right?  It should be `pulumi.runtime.listResourceOutput(undefined, aws.s3.Bucket.isInstance)`?